update-dois/Jenkinsfile

```diff
@@ -66,25 +69,27 @@ pipeline {
 		}
 		// This stage takes the generated report file and sends it to the curator overseeing release.
 		// Before moving onto the next stage of UpdateDOIs, their confirmation that the contents of the report file are correct is needed.
+		/*
 		stage('Send email of updateable DOIs to curator'){
 			steps{
 				script{
 					emailext (
 						body: "This is an automated message. Please review the attached file of Pathway DOIs to be updated and confirm they are correct with the developer running release. Thanks!",
 						to: '$DEFAULT_RECIPIENTS',
-						subject: "UpdateDOIs List for v${env.RELEASE_NUMBER}",
-						attachmentsPattern: "**/doisToBeUpdated-v${env.RELEASE_NUMBER}.txt"
-					)
+						subject: "UpdateDOIs List for v${currentRelease}",
+						attachmentsPattern: "**///doisToBeUpdated-v${currentRelease}.txt" <- This pattern (**/) messses up multi-line comments. When uncommenting, remove 2/3 backslashes before 'doisToBeUpdated'
+		/*			)
```

Comment:
It looks like the `/*` opened on line 72 should still be in effect here?  Is this one needed?

Reply:
Ah okay, I didn't notice that.  Carry on!